### PR TITLE
Processing chain one view

### PR DESCRIPTION
**Pull request by SteRiccio**

### CODE REVIEW

File: webapp/loggedin/modules/analysis/processingChain/processingChainState.js [hunk 1]
```diff
@@ -15,34 +16,47 @@ export const getState = R.pipe(AnalysisState.getState, R.prop(stateKey))
...
-  const chainDirty = R.prop(keys.dirty, state)
-  return R.assoc(keys.orig, chainDirty, state)
+export const appendProcessingStep = step => _updateChainDirty(ProcessingChain.assocProcessingStep(step))
+
+export const dissocStepTemporary = state => {
+  const steps = R.pipe(R.prop(keys.dirty), ProcessingChain.getProcessingSteps)(state)
+  const stepLast = R.last(steps)
+
+  return ProcessingStep.isTemporary(stepLast)
+    ? R.assocPath([keys.dirty, ProcessingChain.keys.processingSteps], R.dropLast(1, steps))(state)
```
Comment (SteRiccio) [inline webapp/loggedin/modules/analysis/processingChain/processingChainState.js]: maybe it's better to not export ProcessingChain.keys and use ProcessingChain.assocProcessingSteps here

Comment (minotogna) [reply webapp/loggedin/modules/analysis/processingChain/processingChainState.js]: i agree, let's do it in the next pull request?

File: webapp/loggedin/modules/home/dashboard/activityLog/activityLogMessageParser/i18nMessageParamsFns/i18nMessageParamsFnsAnalysis.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default {
     }
   },
 
-  [ActivityLog.type.processingStepPropsUpdate]: (survey, i18n) => activityLog => {
+  [ActivityLog.type.processingStepPropUpdate]: (survey, i18n) => activityLog => {
```
Comment (SteRiccio) [inline webapp/loggedin/modules/home/dashboard/activityLog/activityLogMessageParser/i18nMessageParamsFns/i18nMessageParamsFnsAnalysis.js]: check that the message params are generated correctly

File: webapp/loggedin/modules/analysis/processingChain/processingChainState.js (hunk 1 above)
Comment (SteRiccio) [reply webapp/loggedin/modules/analysis/processingChain/processingChainState.js]: ok

**Pull request merged**